charts/rd-professional-api/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "0.1"
 description: Reference data service for professional users
 name: rd-professional-api
-version: 0.0.4
+version: 0.0.5
```

Comment:
0.0.5 is current version..please update version to 0.0.6  and also in flux config